Samples/SendRestExample/SendRestExample/ConnectionStringUtility.cs

```diff
@@ -15,26 +15,34 @@ public ConnectionStringUtility(string connectionString)
             //Parse Connectionstring
             char[] separator = { ';' };
             string[] parts = connectionString.Split(separator);
-            for (int i = 0; i < parts.Length; i++)
+            foreach (var t in parts)
```

Comment:
Convention is to call the element variable `part` if the collection is the plural `parts`.